.github/dependabot.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
```

Comment:
This file seems to be configured incorrectly. Also, I think we do not need to set-up dependabot updates.